highlevel/robot/controller/motion/position.py

```diff
@@ -18,48 +36,66 @@ class PositionController:
     """
     def __init__(self, odometry_function: OdometryFunc,
                  configuration: Configuration, probe: Probe):
+        self.odometry = odometry_function
         self.configuration = configuration
         self.probe = probe
-        self.odometry = odometry_function
 
+        self.distance_travelled: Millimeter = 0.0
```

Comment:
why do you type hint only half of the fields?

fields such as `self.position_right_last` could use a type hint to specify their unit